editor/editor.tsx

```diff
@@ -154,18 +164,23 @@ function Editor(props: any) {
     const urlParams = qs.parse(props.location.search, { ignoreQueryPrefix: true });
     const urlSpec = urlParams?.spec ? JSONCrush.uncrush(urlParams.spec as string) : null;
     const urlGist = urlParams?.gist ?? null;
-    const urlExampleIndex = urlParams?.example
-        ? examples.map(d => d.id).indexOf(urlParams.example as string)
-        : INIT_DEMO_INDEX;
+    const urlExampleId = urlParams?.example as string;
+
+    const defaultCode = urlGist ? emptySpec() : stringify(urlSpec ?? (INIT_DEMO.spec as gosling.GoslingSpec));
 
-    const defaultCode = urlGist
-        ? emptySpec()
-        : stringify(urlSpec ?? (examples[INIT_DEMO_INDEX].spec as gosling.GoslingSpec));
+    // let defaultCode = stringify(INIT_DEMO.spec as gosling.GoslingSpec);
```

Comment:
I was unsure if this is left by mistake, but we can remove this?

Reply:
Sorry, the comment is left here by mistake and I will remove it.